website/templates/emails/reviews_submission_confirmation.txt.mako

```diff
@@ -0,0 +1,31 @@
+Hello ${user.fullname},
+
+% if is_creator:
+    Your ${provider_preprint_word} ${reviewable_title} has been successfully submitted to ${provider.name}.
+% else:
+    ${referrer.fullname} has added you as a contributor to the ${provider_preprint_word} ${reviewable_title} on ${provider.name}, which is hosted on the Open Science Framework: ${reviewable_url}.
+% endif
+
+% if is_pre_moderation:
```

Comment:
Nitpick: Instead of `is_pre_moderation`, I think it would be cleaner to put `workflow` in the context, and do
```mako
% if workflow == 'pre-moderation':
...
% elif workflow == 'post-moderation':
...
% endif
```
Thinking ahead to when we may have more than two workflows